implementations/rust/ockam/ockam_vault_core/src/types.rs

```diff
@@ -26,7 +26,7 @@ cfg_if! {
         pub type SmallBuffer<T> = heapless::Vec<T, U4>;
         /// Buffer for large binaries (e.g. encrypted data). Max size - 512
         pub type Buffer<T> = heapless::Vec<T, U512>;
-        pub type KeyId = heapless::String;
+        pub type KeyId = heapless::String<U32>;
```

Comment:
@mikelodder7 is `U32` the correct heapless generic here?

Reply:
@jared-s It's 32 bytes hex-encoded, so U64 should be good

Reply:
ooops thanks Sanjo, meant to tag you